site/content/en/docs/Guides/feature-stages.md

```diff
@@ -27,7 +27,6 @@ The current set of `alpha` and `beta` feature gates are:
 {{% feature publishVersion="1.9.0" %}}
 | Feature Name | Gate    | Default | Stage | Since |
 |--------------|---------|---------|-------|-------|
-| Multicluster Allocation<sup>*</sup> | N/A | Enabled | `Beta` | 1.6.0 |
```

Comment:
Would it be better to mark this as generally available / stable and leave it in the table instead of removing it?

Reply:
This table seems to be for non-stable features. @markmandel?

Reply:
Yeah, based on our docs you are correct.

For stable features:
"The feature is enabled and the corresponding feature gate no longer exists."